### PR TITLE
fix: remove dpkg-divert from other packages before install

### PR DESCRIPTION
**Pull request by CodeChenL**

### CODE REVIEW

**Review by RadxaYuntian — CHANGES_REQUESTED** [review]
应该用debian replaces & brakes来先移除老版本吧：https://www.debian.org/doc/debian-policy/ch-relationships.html#overwriting-files-in-other-packages

File: debian/radxa-system-config-bookworm.preinst [hunk 1]
```diff
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/bin/bash
```
Comment (RadxaYuntian) [inline debian/radxa-system-config-bookworm.preinst:1]: bashism

**Review by RadxaYuntian — CHANGES_REQUESTED** [review]

File: debian/control [hunk 2]
```diff
@@ -67,8 +67,10 @@ Architecture: all
 Priority: optional
 Replaces: radxa-system-config-common (<< 0.3.15),
           radxa-system-config-amlogic (<< 0.4.0),
+          radxa-system-config-rockchip (<< 0.5.3),
 Breaks: radxa-system-config-common (<< 0.3.15),
         radxa-system-config-amlogic (<< 0.4.0),
+        radxa-system-config-rockchip (<< 0.5.3),
```
Comment (RadxaYuntian) [inline debian/control:73]: rockchip 0.5.2好像没有你那个文件吧？https://github.com/radxa-pkg/radxa-system-config/commits/main/?after=b4422011fab110780da624d711aa416cc398e2aa+34

**Comment by CodeChenL** [comment]
看错包名了https://github.com/radxa-pkg/radxa-system-config/commit/3ad75dbde8357d4d23ffeab45a3708cde6fe7e88#diff-3768c8cf3c7a5c19014f54dcb6860e18a247bcc7890a65cefb3c537391316537，应该是和bullseye互斥就行了

File: debian/control
Comment (RadxaYuntian) [inline debian/control:1]: 这几个suite包都提供了radxa-system-config虚拟包，按照debian policy把缺少的Conflicts和Replaces补上应该可以吧？
https://www.debian.org/doc/debian-policy/ch-relationships.html#replacing-whole-packages-forcing-their-removal

**Review by RadxaYuntian — CHANGES_REQUESTED** [review]

**Review by RadxaYuntian — CHANGES_REQUESTED** [review]
如果可以的话要把其他几个distro的包也加上这两个属性

**Review by RadxaYuntian — CHANGES_REQUESTED** [review]
radxa-system-config-buster也加上吧

**Review by RadxaYuntian — CHANGES_REQUESTED** [review]
rebase

**Review by RadxaYuntian — APPROVED** [review]

**Pull request merged**